src/labels/CustomLabels.labels

```diff
@@ -7327,6 +7327,118 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <protected>true</protected>
         <shortDescription>Title of checklist item FundraisingProcess in Get Started</shortDescription>
         <value>Visualize Your Fundraising Processes</value>
+    </labels>
+      <labels>
+        <fullName>gseuChecklistWorkRecordsDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Work Records in GS for End Users</shortDescription>
+        <value> After you get your constituent and donation data into NPSP, you need to be able to easily find and interact with the records you're working on.</value>
```

Comment:
This change need to be applied everything that a single quote is used in new custom labels in this PR.

```suggestion
        <value> After you get your constituent and donation data into NPSP, you need to be able to easily find and interact with the records you&apos;re working on.</value>
```